docs/en/14-reference/09-support-platform/index.md

```diff
@@ -5,11 +5,11 @@ description: "List of platforms supported by TDengine server, client, and connec
 
 ## List of supported platforms for TDengine server
 
-|                    | **Windows 10/11** | **CentOS 7.9/8** | **Ubuntu 18/20** | **Other Linux** | **UOS** | **Kylin** | **Ningsi V60/V80** | **HUAWEI EulerOS** |
-| ------------------ | ----------------- | ---------------- | ---------------- | --------------- | ------- | --------- | ------------------ | ------------------ |
-| X64                | ●                 | ●                | ●                |                 | ●       | ●         | ●                  |                    |
-| Raspberry Pi ARM64 |                   |                  |                  | ●               |         |           |                    |                    |
-| HUAWEI cloud ARM64 |                   |                  |                  |                 |         |           |                    | ●                  |
+|              | **Windows server 2016/2019** | **Windows 10/11** | **CentOS 7.9/8** | **Ubuntu 18/20** | **统信 UOS** | **银河/中标麒麟** | **凝思 V60/V80** |
```

Comment:
I remember we agreed to remove the Chinese Linux vendor from the English document. However, I don't think to use a Chinese name in an English doc is a good idea.